tests/jballerina-debugger-integration-test/src/main/java/org/ballerinalang/debugger/test/utils/client/connection/TestSocketStreamConnectionProvider.java

```diff
@@ -128,6 +128,11 @@ public void stop() {
         }
     }
 
+    @Override
+    public boolean equals(Object obj) {
+        return super.equals(obj);
+    }
+
```

Comment:
Shall we check equality of `address` and `port` as well?

Reply:
This is done.